.github/workflows/build.yml

```diff
@@ -0,0 +1,27 @@
+name: "Build"
+on:
+  pull_request:
+  push:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - run: |
+        sudo apt-get update
+        sudo apt-get install --yes curl libsndfile1-dev portaudio19-dev
+    - uses: actions/cache@v1
+      with:
+        path: ~/.stack
+        key: ${{ runner.os }}-stack
+    - uses: actions/checkout@v2
+      with:
+        repository: 'soenkehahn/looper'
+        path: 'looper'
+    - run: |
+        cd looper
+        stack install
+        cd ..
+    - uses: actions/checkout@v2
+    - run: |
+        export PATH="$HOME/.local/bin:$PATH"
+        looper run.sh --render output.wav
```

Comment:
I think this file should be named `rc-loops.wav`.

Reply:
Good idea, updated.